compiler/packages/babel-plugin-react-compiler/src/Entrypoint/Program.ts

```diff
@@ -514,43 +486,29 @@ export function compileProgram(
       externalFunctions.push(gating);
     }
 
-    const lowerContextAccess = pass.opts.environment?.lowerContextAccess;
+    const lowerContextAccess = environment.lowerContextAccess;
```

Comment:
Now we're able to rely on the parsed / validated EnvironmentConfig, which comes with default values (instead of using a `PartialEnvironmentConfig` and duplicating defaults